<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrapsMainWindow</class>
 <widget class="QMainWindow" name="CrapsMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rollButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="die1UI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="crapsResources.qrc">:/dieImages/6</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="die2UI">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="crapsResources.qrc">:/dieImages/5</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="currentBankValueUI">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="crapsResources.qrc"/>
 </resources>
 <connections/>
</ui>
